docs/Buttons/SubscribeButton/index.md

```diff
@@ -0,0 +1,85 @@
+## SubscribeButton
+
+Type
+: Function  
+
+Sender
+: SDL
+
+Purpose
+: Subscribe mob app to a particular button
```

Comment:
@KhrystynaDubovyk Could you please correct the typo "mob app"?

Reply:
@ashwink11, please see https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/253/commits/c61ac02ccc8d459f937fbc67c68cf7734ccad09f